bin/manage-worker-types.js

```diff
@@ -0,0 +1,141 @@
+'use strict';
+process.env.DEBUG='';
+
```

Comment:
You should really write some comments..
For bin things that a non-obvious it certainly makes sense at at least add... something like, `/** this utility helps you manage.... .. .. . .. . */`